client/v2/internal/util/util.go

```diff
@@ -24,3 +36,56 @@ func ResolveMessageType(resolver protoregistry.MessageTypeResolver, descriptor p
 
 	return dynamicpb.NewMessageType(descriptor)
 }
+
+// IsSupportedVersion is used to determine in which version of a module / sdk a rpc was introduced.
+// It returns false if the rpc has comment for an higher version than the current one.
+func IsSupportedVersion(input string) bool {
+	return isSupportedVersion(input, buildInfo)
+}
+
+// isSupportedVersion is used to determine in which version of a module / sdk a rpc was introduced.
+// It returns false if the rpc has comment for an higher version than the current one.
+// It takes a buildInfo as argument to be able to test it.
+func isSupportedVersion(input string, buildInfo *debug.BuildInfo) bool {
+	if input == "" || buildInfo == nil {
+		return true
+	}
+
+	moduleName, version := parseSinceComment(input)
+	for _, dep := range buildInfo.Deps {
+		if !strings.Contains(dep.Path, moduleName) {
+			continue
+		}
+
+		if version <= dep.Version {
```

Comment:
We can simplify this and cut out the next 4 lines by doing:

   return version <= dep.version